awx/ui_next/src/screens/Project/ProjectDetail/ProjectDetail.jsx

```diff
@@ -146,6 +156,37 @@ function ProjectDetail({ project }) {
           label={t`Source Control Type`}
           value={scm_type === '' ? t`Manual` : toTitleCase(project.scm_type)}
         />
+        <Detail
+          label={t`Source Control Revision`}
+          value={
+            scm_revision ? (
+              <ClipboardCopy
+                data-cy="project-copy-revision"
+                variant="inline-compact"
+                clickTip={t`Successfully copied to clipboard!`}
+                hoverTip={t`Copy full revision to clipboard.`}
+                onCopy={event => {
+                  const clipboard = event.currentTarget.parentElement;
+                  const el = document.createElement('textarea');
+                  el.value = scm_revision.toString();
+                  clipboard.appendChild(el);
+                  el.select();
+                  document.execCommand('copy');
+                  clipboard.removeChild(el);
+                }}
```

Comment:
`execCommand` is deprecated ([apparently](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand))

I think we can do this now:

```suggestion
                onCopy={() => navigator.clipboard.writeText(scm_revision.toString())}
```

Reply:
👍 will check that out.  I actually just yoinked this code from the PF React repo: https://github.com/patternfly/patternfly-react/blob/master/packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx#L13-L21.  Seems like they might need an update too.